### ISSUE TITLE
Feature: Actual versions docker images on docher hub

### ISSUE THREAD

**Issue by 4n70w4** [Aidaho12/haproxy-wi#241]
Hi! I could not find on the hub docker images of this project.
It would be convenient to have an images with actual versions on a docker hub.

**Comment by Aidaho12** [comment]
Hi!

There is docker image, but not in docker hub: https://haproxy-wi.org/docker.py

**Comment by 4n70w4** [comment]
Thanks for link.

Quote from post https://pikabu.ru/story/kak_sluchayno_napisat_webgui_dlya_haproxy_7648038

> это единственный бесплатный GUI

Sorry, I can't try your product yet.

**Comment by Aidaho12** [comment]
Yep, it's free, but if you don't want or you can't build your own image donate a little money for supporting the project 😉

**Comment by 4n70w4** [comment]
I just wanted to try and see if I needed this GUI and will it be useful to me. Build images self is too difficult for this. Forced to refuse = (

**Comment by Aidaho12** [comment]
There is demo site. Or if you want to try exactly the image I can provide trial period for you. Image continue working when trial will end

**Comment by Aidaho12** [comment]
Well, you can download docker image without subscription now :)
